### DISCUSSION TITLE
Moving from cloud to local and vice versa

### DISCUSSION THREAD

**Discussion by olup** [pulumi/pulumi#14419] [General]
I can't find documentation on how to leave the pulumi cloud state management to a local (gcp based) state backend and keeping my stack with me. I tried exporting and importing but the stack indicates some secret manager service that relies on pulumi cloud. Anyone with pointers ?

**Comment by Frassle** [comment]
You need to decrypt everything with `pulumi stack export --show-secrets`, or first change the stack to a different secret provider that's also supported in the local backend (such as passphrase or kms) with `pulumi stack change-secrets-provider` before exporting.

**Reply by olup** [reply]
show-secrets seems to still rely on the secret provider
